<launch>
  <node name="dispatcher" pkg="bwi_dispatcher" type="dispatcher" output="screen">
    <param name="status_topic" value="/machine" />
    <param name="max_status_duration" value="20" />
    <param name="publish_rate" value="10" />
  </node>
</launch>
